java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/deploy/TagDeployCommand.java

```diff
@@ -86,20 +85,25 @@ public void run() {
 
             var deploymentPath = Path.of(artifactDirectory);
 
+            // TODO: ### To discuss: I don't think we need this. It was a *secondary* write of
+            //       source - writing after preprocess, but including any pre-build changes. There
+            //       is also the difficulty that we don't pass the hash we wrote through. Although
+            //       I think we can grab is from the GitArchive
+            //
             // TODO: Should we write out to a 'DependencyPipelineResults' a GitArchive?
-            Git.GitStatus archivedSourceTags = new Git.GitStatus();
-            // Save the source first regardless of deployment checks
-            if (isNotEmpty(gitIdentity) && gitToken.isPresent()) {
-                var git = Git.builder(gitURL, gitIdentity, gitToken.get(), gitDisableSSLVerification);
-                if (reuseRepository) {
-                    git.initialise(scmUri);
-                } else {
-                    Log.warnf("Not reusing repository; creating under %s", scmUri);
-                    git.create(scmUri);
-                }
-                Log.infof("Pushing changes back to URL %s", git.getName());
-                archivedSourceTags = git.add(sourcePath, commit, imageId);
-            }
+//            Git.GitStatus archivedSourceTags = new Git.GitStatus();
+//            // Save the source first regardless of deployment checks
+//            if (isNotEmpty(gitIdentity) && gitToken.isPresent()) {
+//                var git = Git.builder(gitURL, gitIdentity, gitToken.get(), gitDisableSSLVerification);
+//                if (reuseRepository) {
+//                    git.initialise(scmUri);
+//                } else {
+//                    Log.warnf("Not reusing repository; creating under %s", scmUri);
+//                    git.create(scmUri);
+//                }
+//                Log.infof("Pushing changes back to URL %s", git.getName());
+//                archivedSourceTags = git.add(sourcePath, commit, imageId);
+//            }
```

Comment:
@vibe13 To discuss ; I don't think we need the Git push changes here (they are after the preprocess changes (which are also stored in git) but also include any pre-build script changes.

Reply:
I think this is one I wanted to discuss offline. It relates to [this](https://docs.google.com/document/d/1qJ0rCdBs3iB0ETkxxNaIqRxYpoiMmpcVzPCgheaotgw/edit#bookmark=id.lis275csct3t) - specifically storing sources after preprocess and after pre-build but before build. PNC doesn't do that and doesn't store any changes user's script so I think we can drop it.